skimage/segmentation/_watershed.py

```diff
@@ -26,7 +26,7 @@
 
 
 def _validate_inputs(image, markers, mask, connectivity):
-    """Ensure that all inputs to watershed have matching shapes and types.
+    """Ensures that all inputs to watershed have matching shapes and types.
```

Comment:
Please revert this change — see [PEP-257](https://peps.python.org/pep-0257/) for docstring guidelines.

Reply:
I did attempt to fix the bug the ideal way as suggested by @stefanv. However, I got a `ValueError` from one of the Cypthon files. Thanks for pointing me to a resources that should help.
I will revert the changes and attempt the fix the bug appropiately

Reply:
Don't forget to revert this change!

Reply:
Sure.